SPECS-EXTENDED/gdisk/gdisk.spec

```diff
@@ -49,10 +47,40 @@ make test
 %{_mandir}/man8/fixparts.8*
 
 %changelog
+* Wed Nov 06 2024 Durga Jagadeesh Palli <v-dpalli@microsoft.com> - 1.0.10-3
```

Comment:
Changelog entries are also modified heavily. As this is upgrade we need to have only one entry after the entry dated "* Wed Mar 08 2023"

Reply:
Change log has been updated as you suggested.